web/hooks.go

```diff
@@ -234,8 +234,13 @@ func runTests(logger *zap.SugaredLogger, db database.Database, runner ci.Runner,
 		approve = lastSubmission.GetApproved()
 	}
 
-	// also approved if autoapprove is on and total score is above 80%
-	if selectedAssignment.AutoApprove && result.TotalScore() >= 80 {
+	// if auto approve is on, use lower limit from yaml. If not set, use default value
```

Comment:
I think this comment is a bit inaccurate/unclear. Here is my understanding:
```
If AutoApprove {
  if score limit defined in yaml file {
    use yaml defined score limit 
  } else {
    use default score limit
  }
} else {
  // manual approval required
}
```
If the above is correct understanding, maybe the following comment could be used:
```
// for auto approve, use score limit from yaml file or default value if score limit not defined
```
or 
```
// for auto approve, use default score limit unless defined in yaml file 
```
Or do we also use the score limit to color mark whether or not an assignment is "manually approvable"??

Reply:
We don't. Should we?

Reply:
I think we discussed it, or maybe I discussed it in my own head :-)

But it can be part of a different PR if we decide to do it...

Reply:
Yeah, it totally can be done. I'll push it here when ready, as it seems to belong here best.

Reply:
Ready now.